sources/styles.css

```diff
@@ -147,3 +143,296 @@ div.button i.fa-arrow-down {
+  padding-top: 100px;
+  width: 100%;
+  justify-content: center;
+}
+
+div.work-title {
+  text-align: center;
+}
+
+div.work-title h2 {
+  font-family: "Inter", sans-serif;
+  margin-top: 40px;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 700;
+  margin-bottom: 90px;
+  line-height: 52px;
+  color: #172b4d;
+  box-sizing: border-box;
+}
+
+article.project-list {
+  margin-bottom: 12px;
+  box-sizing: border-box;
+  border: 1px solid #acb7c3;
+  border-radius: 8px;
+  display: flex;
+  width: 100%;
+  justify-content: center;
+}
+
+div.project1 img {
+  height: 237px;
+  width: 100%;
+}
+
+div.project1 h2 {
+  text-align: center;
+  width: 295px;
+  height: 57px;
+  margin-bottom: 16px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+}
+
+ul.button-grid {
+  width: 100%;
+  display: flex;
+  align-content: flex-start;
+  justify-content: space-evenly;
+  align-self: center;
+}
+
+ul.button-grid li.lang-button1 {
+  display: flex;
+  width: 103px;
+  height: 32px;
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+ul.button-grid li.lang-button2 {
+  display: flex;
+  width: 45px;
+  height: 32px;
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+ul.button-grid li.lang-button3 {
+  display: flex;
+  width: 82px;
+  height: 32px;
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+ul.button-grid li.lang-button4 {
+  display: flex;
+  width: 51px;
+  height: 32px;
+  list-style: none;
+  box-sizing: border-box;
+  border: 1px solid #ebf0eb;
+  background-color: #ebf0eb;
+  border-radius: 4px;
+  padding-top: 8px;
+  padding-right: 12px;
+  padding-bottom: 8px;
+  padding-left: 12px;
+  font-family: "Inter", sans-serif;
+  font-weight: 600;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 3%;
+}
+
+section.main-button {
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  align-self: center;
+}
+
+button.projectview {
```

Comment:
- [ ] Please make the necessary changes to match the design, like removing the back border
Yours:
![image](https://user-images.githubusercontent.com/73140461/148589994-a568f345-e846-4e58-8bd7-c555de3e5fe4.png)
Design:
![image](https://user-images.githubusercontent.com/73140461/148590168-98b0debd-6ea1-4a6a-8ef5-e1b525ddef19.png)